docs/styling.md

```diff
@@ -0,0 +1,119 @@
+
+To do that, you use the `createTheme` function imported from `@vtex/store-ui`.
+
+>ℹ️ `@vtex/store-ui` is our plugin that exports common ecommerce components
+
+### Modifying or adding new styles to themes
+To style a pre-existing plugin, you use the gatsby functionality of [theme shadowing](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/).
+
+Suppose you want to alter the color of the `searchSuggestions` add to cart button component from primary to secondary.
+
+ <img src="../images/suggestions.png">.
+
+You would need to find the correspondent for this button and alter the styles using theme shadowing. For the case of search suggestions, the theme for this component is located at this [file](https://github.com/vtex/faststore/blob/master/packages/gatsby-theme-store/src/components/SearchSuggestions/theme.ts).
+
+```ts
+// before altering the color
+    products: {
+      width: 'inherit',
+
+      button: {
+        backgroundColor: 'primary',
+      },
+
+      title,
+
+      list: {
+        ...list,
+        display: 'flex',
+        flexWrap: 'nowrap',
+      },
+
+      total: {
+        paddingTop: '10px',
+        color: 'text',
+        textDecoration: 'underline',
+        textAlign: 'center',
+        cursor: 'pointer',
+        width: '100%',
+        backgroundColor: 'white',
+      },
+    },
+```
+
+To alther the color of the button, you need to change the `backgroundColor` property from `primary` to `secondary`.
+
+```
+// after altering the color
+    products: {
+      width: 'inherit',
+
+      button: {
+        backgroundColor: 'secondary',
+      },
+
+      title,
+
+      list: {
+        ...list,
+        display: 'flex',
+        flexWrap: 'nowrap',
+      },
+
+      total: {
+        paddingTop: '10px',
+        color: 'text',
+        textDecoration: 'underline',
+        textAlign: 'center',
+        cursor: 'pointer',
+        width: '100%',
+        backgroundColor: 'white',
+      },
+    },
+```
+
+## Remarks
+It is important to understand that theme shadowing only occurs on `@vtex/gatsby-plugin-theme-ui/index`. We structure the code with multiple files to improve readability. But the plugin only considers what is exported on  `index.ts`. Every theme from the components should be an argument for the `createTheme` function.
+
+Example:
+
+```ts
+// @vtex/gatsby-plugin-theme-ui/index.ts
+export default createTheme(
+  base,
+  breadcrumb,
+  minicart,
+  infoCardTheme,
+  headerTheme,
+  productQuantityTheme,
+  minicartTheme,
+  searchControlsTheme,
+  searchSuggestionsTheme,
+  searchBarTheme,
+  searchTheme,
+  custom,
+  sliderTheme,
+  offerTheme,
+  productSummaryTheme,
+  productDetailsTheme,
+  loginTheme,
+  authTheme
+)
+```
+
+## Adding styles to a react component
+To add new styles from a theme into a react component, you have to discover what is the name of the variant that
```

Comment:
Discover what is the name of the variant that uses the theme, than change it to de desired one.
 
 (Not sure this is it, but...)